informant.gemspec

```diff
@@ -11,4 +11,6 @@ Gem::Specification.new do |s|
   s.description = "Informant is a full-featured form builder for Ruby on Rails which promotes a simple syntax that keeps your views clean. Everything about a field (label, description, error display, etc) is encapsulated in a single method call."
   s.files       = `git ls-files`.split("\n") - %w[informant.gemspec Gemfile]
   s.require_paths = ["lib"]
+  s.add_dependency('actionview', '>= 4.0')
```

Comment:
This will break compatibility for people currently using Informant with Rails <4.0. Is it possible to make the necessary changes for 4.0 without breaking anyone's apps? We can proceed regardless, because 3.0 is old, but it will mean a different type of release.